service/src/main/java/org/eclipse/keti/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -280,7 +280,12 @@ private void logSetEntityTimestampsDebugMessage(final String timestamp, final St
 
     @Override
     public void resetForPolicySet(final String zoneId, final String policySetId) {
+        // Well-known internal policy set name is used to construct a decision cache key in case of a single policy
+        // set. To ensure correct invalidation of stale cached decision, invalidate cache timestamp for both specified
+        // policy set name and a well-known internal policy set name regardless if present or not.
         resetForEntity(zoneId, policySetId, EntityType.POLICY_SET, AbstractPolicyEvaluationCache::policySetKey);
+        resetForEntity(zoneId, PolicySet.INTERNAL_POLICY_SET_NAME, EntityType.POLICY_SET,
```

Comment:
A more accurate way to represent this key is probably `ZONE_POLICY_SETS`.  So when any policy set is updated, the `ZONE_POLICY_SETS` key is also updated. Is that better?

The comment on line 283 can be removed, and the behavior needs to be explained in the evaluation service.

Reply:
I am proposing a change in the name of the Key not EntityType. Per our discussion: constant name: `POLICY_SET_ANY` value could be `ANY`